### PR TITLE
Update 508-accessibility-best-practices.md

### PR DESCRIPTION
**Pull request by jenstrickland**
Supplemented with additional details for testing and tooling; a future iteration will supplement with product strategy, user research, design, and qa best practices.

@1Copenut and @rianfowler — would you please review for accuracy?

### CODE REVIEW

**Review by rianfowler — COMMENTED** [review]
looks good to me but I have to defer to trevor for approval on some of the finer points about testing.

File: platform/accessibility/508-accessibility-best-practices.md [hunk 1]
```diff
@@ -35,22 +42,26 @@
...
-   3. **Each rendered page must pass an axe check.**
+- **axe Scans** Front-end engineers should install the [axe plugin for Chrome or Firefox](https://deque.com/axe) and [run automated scans](https://www.youtube.com/watch?v=FW1giWW5M9I) periodically during their daily work against all application pages or states. The axe-core plugin is available for [Chrome](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) and [Firefox](https://addons.mozilla.org/en-US/firefox/addon/axe-devtools/). This adds a tab to developer tools and provides on-demand accessibility scanning for single pages during development.
+- Going forward, we should also be testing pages with [axe-coconut](https://www.deque.com/blog/test-leading-edge-accessibility-axe-coconut-axe-core-3-0/), which identifies experimental WCAG2 warnings and best practices.
+- Run the [axe-end to end script with CSV and console output](https://github.com/adhocteam/a11y-snippets/tree/master/test-scripts/axe-e2e) for automated scans of high page count static sites. Errors can be outputted to CSV for easy spreadsheet import.
```
Comment (1Copenut) [inline platform/accessibility/508-accessibility-best-practices.md]: Let's remove this bullet point for now. The axe e2e script is way out of date and needs to be brought up to the new version of axe and Chrome Driver.

File: platform/accessibility/508-accessibility-best-practices.md [hunk 2]
```diff
@@ -35,22 +42,26 @@
...
+      * Client-side applications will be tested for accessibility as part of their larger e2e test suite.
+      * Client-side applications **must include an axe check** in their end-to-end tests. The engineering team has created a [Nightwatch helper function](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/testing/e2e/nightwatch-commands/axeCheck.js) for this purpose
+      * ESLint exists as a Node module with a command-line interface. It runs a lint scan on all components as a pre-commit hook, and will break on errors.
+      * Generally speaking, ESLint and the needed dependencies should be installed when users *yarn install* from the *vets-website* root.
+      * If a build error occurs, fix the issue and submit your code again.
 
-3. If a build error occurs, fix the issue and submit your code again.
-
-## Manual Accessibility/508 Testing
+## Manual Accessibility Testing
```
Comment (1Copenut) [inline platform/accessibility/508-accessibility-best-practices.md]: The table of contents link at the top of the page needs to be updated so this works properly. The text should match this heading, and the link link will look like this:

```html
[Manual Accessibility Testing](#manual-accessibility-testing)
```

**Review by 1Copenut — CHANGES_REQUESTED** [review]

**Comment by jenstrickland** [comment]
Changes accepted. Thank you!

**Pull request merged**